### DISCUSSION TITLE
In Nuxt3, we don't need to specify plugins in `nuxt.config.ts` specifically

### DISCUSSION THREAD

**Discussion by anburocky3** [iendeavor/pinia-plugin-persistedstate-2#21] [General]
From the docs, I can see that, for nuxt3, we have to import the plugin, but `nuxt3` will automatically include any plugins within the `plugins` folder right?

Without having it also doesn't raise any problem. Then why it is needed?

```ts
plugins: ['@/plugins/persistedstate.universal.ts'],
```

**Comment by soc221b** [comment]
If nuxt3 does, you don't have to explicitly include it in nuxt configuration. 🥳 

(Nuxt 3 is still in beta, these features may also change in the future, the documentation will be updated when the rc arrives.)
